Fix broken / incorrect links (#531)
MicrosoftDocs/WindowsCommunityToolkitDocs: docs/controls/ImageEx.md

> ImageEx control supports use of Progress Indicator. This can be enabled by adding ImageEx template from previous release of the control.
> 

Reviewer: ```suggestion
```

ImageEx control supports use of Progress Indicator. This can be enabled by adding ImageEx template from previous release of the control.

> 

Reviewer: ```suggestion
```

ImageEx control supports use of Progress Indicator. This can be enabled by adding ImageEx template from previous release of the control.

> - [ImageEx Control with Progress Indicator XAML File](https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/rel/2.2.0/Microsoft.Toolkit.Uwp.UI.Controls/ImageEx/ImageEx.xaml) is the XAML template used in v.2.2.0 of toolkit.

Reviewer: ```suggestion
```

Not sure why we removed the progress ring in the past, but if we don't support the feature anymore, we should just remove this from the docs entirely.,

> - [ImageEx Control XAML File](https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/main/Microsoft.Toolkit.Uwp.UI.Controls.Core/ImageEx/ImageEx.xaml) is the XAML template used in the toolkit for the default styling.

Reviewer: ```suggestion
- [ImageEx Control XAML File](https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/rel/7.0.0/Microsoft.Toolkit.Uwp.UI.Controls.Core/ImageEx/ImageEx.xaml) is the XAML template used in the toolkit for the default styling.
```

We still want to point to a specific version so when we move things in the future (if we do) the links don't break. These links were just missed in the last refactor so were still broken.
